.github/ISSUE_TEMPLATE/issue_with_content.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Lesson content issue template
+about: Let us about an issue you found with the material
+title: ''
+labels: 
+assignees: ''
+
+---
+
+<details>
+<summary><strong>Instructions</strong></summary>
+
+Thanks for taking the time to report an issue with the lesson! :heart:
+
+Please let us know about:
+
+- Technical issues or out of date screenshots and package/python versions
+- Inconsistencies or errors in the material
+- Missing references or links to external resources
+- How material can be reworded to avoid confusion
+- Any other issues
```

Comment:
```suggestion
- Missing references or links to external resources
- How material can be reworded to avoid confusion
- Problems with building or rendering the lesson material locally, with a concise step-by-step procedure to replicate the issue
- Any other issues
```